python/ray/services.py

```diff
@@ -1043,6 +1085,43 @@ def start_raylet(redis_address,
     return process_info
 
 
+def build_java_worker_command(java_worker_options, redis_address,
+                              plasma_store_name, raylet_name):
+    """This method assembles the command used to start a Java worker.
+
+    Args:
+        java_worker_options (str): The command options for Java worker.
+        redis_address (str): Redis address of GCS.
+        plasma_store_name (str): The name of the plasma store socket to connect
+           to.
+        raylet_name (str): The name of the raylet socket to create.
+
+    Returns:
+        The command string for starting Java worker.
+    """
+
+    if java_worker_options is None:
+        raise Exception("java_worker_options must not be "
+                        "None if include_java is True.")
```

Comment:
nit, it's okay to just do `assert java_worker_options is not None`. Because we have a default value, it shouldn't be None unless there's a bug.